lib/components/error_dialog.dart

```diff
@@ -8,16 +8,22 @@ class ErrorDialog extends StatelessWidget {
   final String details;
   final VoidCallback? onRightButtonPressed;
 
```

Comment:
Please set line length in your editor to 120 so we don't get a bunch of line break changes. Thanks